### ISSUE TITLE
Samsung Galaxy Note 10+ (snapdragon) stock cam doesn't generate proofs, and sharing proofs crashes

### ISSUE THREAD

**Issue by shayaknyc** [guardianproject/proofmode#79]
Just updated to the latest release apk on my phone. Needed to force stop, clear cache and clear data to get it to even function. Doesn't detect photos taken with the phone's stock camera app, when sharing the photo just taken from stock cam with Proofmode, it says no proof generated. If I take a photo with a 3rd party app, and share the photo from stock gallery to proofmode, it gives me the option for basic or robust, tapping on either the first time, nothing happens, if I try to share the same photo again to proofmode, it gives me the option again for basic or robust, but on the 2nd attempt, shows that the app FC'd. Any ideas?

**Comment by n8fr8** [comment]
New release for testing: https://github.com/guardianproject/proofmode/releases/tag/0.0.15-BETA-1

**Comment by shayaknyc** [comment]
Ok, far more stable now. Can at least force generate a proof and can send basic and robust. However, there doesn't appear to be an actual proof generated when the photo is taken, only when tapping the generate button. This feels like it's not working properly......

**Comment by n8fr8** [comment]
Yes, there seems to be an issue with the Galaxy devices in general, regarding the callback/event notification to ProofMode of when new photos and video are available. Working on it...

**Comment by shayaknyc** [comment]
Cool, thanks for the update! 

I'm no developer, but I think I read somewhere in the changelog that the image/video detection is based on scanning the DCIM folder? If that's true, I wonder if it's related to requesting extended storage permissions?

**Comment by n8fr8** [comment]
That is one technique that is used. There is another where the OS itself is supposed to trigger "jobs" to process new media being available, but it seems like one of those features that doesn't work as expected across different flavors of Android.

**Comment by shayaknyc** [comment]
Yeah, short of a wholesale reboot of the phone, the new media trigger rarely works on my samsung device. For example, if I used termux to download a video using yt-dlp (the fork of youtube-dl), I have to MOVE (not copy) the file to another directory using a file manager to "trigger" the scan of new media. IDK what's up with Samsung specifically, but that trigger rarely works.....

**Comment by n8fr8** [comment]
Fixed with Proofmode v3 and built in camera
